tests/features/test_velocity.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from databallpy.features.velocity import get_velocity
+
+
+class TestGetVelocity(unittest.TestCase):
+    def setUp(self):
```

Comment:
Maybe add a little more variation in locations, add negative values and also add some `np.nan` values to see if it handles them properly.